index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="calc">
+        <div class="calc__body"></div>
+        <output class="calc__output"></output>
+        <button class="calc__button calc__number-one" id="numberOne"><input type="image" src="images/1.jpg"></button>
```

Comment:
Lets use <img> instead of <input> for all buttons since it is semantically incorrect to is input in this case (imho)